config.json

```diff
@@ -45,6 +45,18 @@
         "prerequisites": [],
         "status": "active"
       },
+      {
+        "slug": "annalyns-infiltration",
+        "name": "Annalyn's Infiltration",
+        "uuid": "8833b0c0-1008-4426-8225-1dc2b4852d52",
+        "concepts": [
+          "booleans"
+        ],
+        "prerequisites": [
+          "basics"
+        ],
+        "status": "active"
```

Comment:
What state should all the concept exercises have? The other two exercises have "active". I would personally make all concept exercises "beta" for the first few weeks of v3.

> beta: This signifies active exercises that are new and which we would like feedback on. We show a beta label on the site for these exercise, with a Call To Action of "Please give us feedback."

Reply:
I agree, "beta" is the best option for now. I'd be glad to adjust these in a separate PR, or feel free to make the changes on your branch if you'd like.

Reply:
I'll make the change in this PR